src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/KmerAndCount.java

```diff
@@ -18,7 +18,7 @@
 
     public KmerAndCount( final SVKmerLong kmer ) { this(kmer,1); }
 
-    KmerAndCount( final SVKmerLong kmer, final int count ) {
+    public KmerAndCount( final SVKmerLong kmer, final int count ) {
```

Comment:
Since is public, must it check that the ```kmer``` is not null and ```count``` is 0 or positive?

Reply:
added a check for non-negative number.
But for the null check, I don't quite understand yet what the benefit is for this specific one. It delegates to the parent class's ctor, which would immediately throw an NPE if it null. An explicit null check would pretty much to the same thing, right?

Reply:
As per GATK3  GATK4 policy NPE are not acceptable; you can argue that with @droazen but I think is a lost cause, right David?

Reply:
added null check